vyper/types.py

```diff
@@ -180,22 +180,6 @@ def canonicalize_type(t, is_indexed=False):
     raise Exception("Invalid or unsupported type: " + repr(t))
 
 
-def parse_abi_type(t):
```

Comment:
Why remove this?

Reply:
Doesn't get used anywhere, and would be super easy to add if we ever need it.